package.json

```diff
@@ -21,17 +21,24 @@
     "@patternfly/react-core": "^1.35.0",
     "@types/jest": "^23.3.9",
     "babel-loader": "^8.0.4",
+    "clean-webpack-plugin": "^1.0.0",
```

Comment:
Could we get by with not using `clean-webpack-plugin`? See this discussion

https://github.com/johnagan/clean-webpack-plugin/issues/75
https://github.com/johnagan/clean-webpack-plugin/issues/60

> At this point, I'd rather a more popular clean library take over, rather than trying to keep up with another or solve for every use-case.

I've used rimraf in the past, works great and it's not tied directly to webpack which is nice. You could add a clean target like `"clean": "yarn rimraf dist"` and just call that before running the build script.